### PR TITLE
Release 2.4.5

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by ktoso** [comment]
You have to also update `_includes/footer.html`

**Comment by ktoso** [comment]
LGTM otherwise

File: news/_posts/2016-05-17-akka-2.4.5-released.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Today, we announce that the API has been completed, and except minor changes we do not expect to do any large
+refactorings like this one – we’re on our road to a stable Routing API. This sadly means that the changes are
+rather wide-spread, and migrating from old to new may feel like learning an entire new API (which it is). If you run
+into trouble migrating contact us via the mailing list or gitter chat; customers may of course ask for help via the
+Lightbend support portal.
+
+You will notice that [directives documentation](http://doc.akka.io/docs/akka/2.4.5/java/http/routing-dsl/directives/alphabetically.html)
+is complete, however lacking code examples in some cases. We would like to ask for your help in finishing
+the docs examples by adding code examples–these efforts can be coordinated as part of the [#20466](https://github.com/akka/akka/issues/20466) ticket.
```
Comment (ktoso) [inline news/_posts/2016-05-17-akka-2.4.5-released.md]: maybe code examples => code snippets
examples is used a lot in this sentence already

Comment (johanandren) [inline news/_posts/2016-05-17-akka-2.4.5-released.md]: or just "... ask for your help in finishing the docs by adding code examples ..." ?

Comment (ktoso) [inline news/_posts/2016-05-17-akka-2.4.5-released.md]: good

**Comment by johanandren** [comment]
@patriknw @drewhk @2m do I hear another LGTM coming?

**Comment by patriknw** [comment]
LGTM

**Pull request merged**